code/datums/uplink_item.dm

```diff
@@ -1351,7 +1351,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 /datum/uplink_item/device_tools/medgun
 	name = "Medbeam Gun"
-	desc = "Medical Beam Gun, useful in prolonged firefights."
+	desc = "Medical Beam Gun, useful in prolonged firefights. DO NOT CROSS THE BEAMS. Crossing the beams or attaching two beams to one target will have explosive consequences."
```

Comment:
I'd recommend rewording slightly to indicate this only occurs when another medbeam gun is used:
`"DO NOT CROSS THE BEAMS. Crossing beams with another Medbeam Gun or attaching two beams to one target will have explosive consequences."`

Reply:
IT IS DONE.